<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcePopUp</class>
 <widget class="QWidget" name="AcePopUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New ACE</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="Username_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Username :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Username">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="AccessRightsLayout">
         <item>
          <widget class="QLabel" name="AccessRightsLabel">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Access Rights :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_GENERIC_READ">
           <property name="text">
            <string>GENERIC_READ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_WRITE_OWNER">
           <property name="text">
            <string>GENERIC_WRITE</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="tristate">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_SYNCHRONIZE">
           <property name="text">
            <string>GENERIC_EXECUTE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_STANDARD_RIGHTS_REQUIRED">
           <property name="text">
            <string>GENERIC_ALL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_STANDARD_RIGHTS_READ">
           <property name="text">
            <string>DELETE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_STANDARD_RIGHTS_WRITE">
           <property name="text">
            <string>READ_CONTROL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_STANDARD_RIGHTS_EXECUTE">
           <property name="text">
            <string>WRITE_DAC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_STANDARD_RIGHTS_ALL">
           <property name="text">
            <string>WRITE_OWNER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_SPECIFIC_RIGHTS_ALL">
           <property name="text">
            <string>SYNCHRONIZE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_GENERIC_WRITE">
           <property name="text">
            <string>STANDARD_RIGHTS_REQUIRED</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_GENERIC_EXECUTE">
           <property name="text">
            <string>STANDARD_RIGHTS_READ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_GENERIC_ALL">
           <property name="text">
            <string>STANDARD_RIGHTS_WRITE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_DELETE">
           <property name="text">
            <string>STANDARD_RIGHTS_EXECUTE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_READ_CONTROL">
           <property name="text">
            <string>STANDARD_RIGHTS_ALL</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="_WRITE_DAC">
           <property name="text">
            <string>SPECIFIC_RIGHTS_ALL</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="AceTypeGroupBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="title">
          <string>ACE Type </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="AccessAllowed">
            <property name="text">
             <string>Access Allowed</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="AccessDenied">
            <property name="text">
             <string>Access Denied</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
